addons/account_peppol/views/res_partner_views.xml

```diff
@@ -59,7 +59,7 @@
         <field name="state">code</field>
         <field name="code">
             for record in records:
```

Comment:
It's a bit of a shame that it is a `ensure_one` function because otherwise we could have returned an action (just in case) but no need to change that now...

Just a bad design that I see too often.